crew

```diff
@@ -10,7 +10,7 @@ require 'fileutils'
 @pkgName = ARGV[1]
 
 ARCH = `uname -m`.strip
-ARCH_LIB = if ARCH == 'x86_64' then 'lib64' else 'lib' end
+ARCH_LIB = if ARCH == 'x86_64' or ARCH == 'aarch64' then 'lib64' else 'lib' end
```

Comment:
Is this modification required for OnHub?

For the case of chromebook, aarch64 is 64 bit kernel but 32 bit userland.  So, we need to use `lib` for aarch64.